public/coda/coding_progress.html

```diff
@@ -27,28 +28,34 @@
         };
         firebase.initializeApp(config);
     </script>
+
 </head>
 <body>
-    
+<nav class='nav'>
+    <div class="nav-wrapper">
+        <a href="https://www.africasvoices.org/wp-content/themes/africa/static/img/africas-voices-logo-pad.svg" class="brand-logo center">Operations Dashboard</a>
```

Comment:
This looks a bit odd. Could you elaborate on what you were aiming to achieve with the <a> tag referencing an image as its target?

Reply:
I wanted to include the AVF logo with the dashboard title. Have fixed it 
![image](https://user-images.githubusercontent.com/23135322/49067066-028fcd80-f234-11e8-8bf7-4336e6516898.png)